src/store/api.js

```diff
@@ -308,6 +309,6 @@ export async function validateMigrations(migrations) {
 }
 
 export async function getTop3Events() {
-  const res = await fetch(`${POAP_API_URL}/top-3-events`)
+  const res = await fetch(`${POAP_API_URL}/top-3-events`, {headers: {'X-API-Key': POAP_API_API_KEY}})
```

Comment:
Shouldn't we do a wrapper that always injects the API key and sets the base URL, so it's clean?